package.json

```diff
@@ -13,7 +13,7 @@
     "test": "npm run test:prefixed-links && npm run test:internal-links && npm run test:html-validation && npm run test:spelling",
     "test:html-validation": "html-validate _site/**/*.html",
     "test:internal-links": "node check-links.js",
-    "test:prefixed-links": "! (grep -Erl \"\\(/|href=['\\\"]/\" pages && echo \"ERROR: Internal links must use {% page \"page name\" %} to work correctly with Cloud.gov Pages previews. Fix the above pages.\")",
+    "test:prefixed-links": "! (grep -Erl \"\\]\\(/|href=['\\\"]/\" pages && echo \"ERROR: Internal links must use {% page \"page name\" %} to work correctly with Cloud.gov Pages previews. Fix the above pages.\")",
```

Comment:
This change tightens up the regex a little bit to look for `](/` to find relative links that are not using the `{% page %}` shortcode. The need comes from the case where we need to put file paths inside parentheses. For example:

```
Put the file in the magic directory (/Users/yourusername/.ssh)
```

The current regex flags that line as an invalid link, but it is, of course, not a link at all. Adding the preceding closing square bracket to the regex allows this line to pass while continuing to enforce the intent of the check.